### PR TITLE
Tweaking UI

### PR DESCRIPTION
**Pull request by DuckSoft**

### CODE REVIEW

File: src/ui/w_SubscriptionManager.cpp [hunk 1]
```diff
@@ -9,10 +9,9 @@ SubscribeEditor::SubscribeEditor(QWidget *parent) : QDialog(parent)
     setupUi(this);
     QvMessageBusConnect(SubscribeEditor);
     UpdateColorScheme();
+    adjustSize();
     for (auto subs : ConnectionManager->Subscriptions())
-    {
-        subscriptionList->addTopLevelItem(new QTreeWidgetItem(QStringList{ GetDisplayName(subs), subs.toString() }));
-    }
+    { subscriptionList->addTopLevelItem(new QTreeWidgetItem(QStringList{ GetDisplayName(subs), subs.toString() })); }
     if (subscriptionList->topLevelItemCount() > 0)
```
Comment (Qv2ray-dev) [inline src/ui/w_SubscriptionManager.cpp]: I wonder why things change like this.

File: src/ui/w_MainWindow.cpp [hunk 2]
```diff
@@ -867,10 +863,7 @@ void MainWindow::OnGroupCreated(const GroupId &id, const QString &displayName)
 }
 void MainWindow::OnGroupDeleted(const GroupId &id, const QList<ConnectionId> &connections)
 {
-    for (auto conn : connections)
-    {
-        groupNodes.value(id)->removeChild(connectionNodes.value(conn).get());
-    }
+    for (auto conn : connections) { groupNodes.value(id)->removeChild(connectionNodes.value(conn).get()); }
```
Comment (Qv2ray-dev) [inline src/ui/w_MainWindow.cpp]: See below

File: src/ui/w_MainWindow.cpp [hunk 3]
```diff
@@ -259,10 +258,7 @@ MainWindow::MainWindow(QWidget *parent) : QMainWindow(parent)
         MWAddGroupItem_p(group);
         auto connections = ConnectionManager->Connections(group);
 
-        for (auto connection : connections)
-        {
-            MWAddConnectionItem_p(connection, group);
-        }
+        for (auto connection : connections) { MWAddConnectionItem_p(connection, group); }
```
Comment (Qv2ray-dev) [inline src/ui/w_MainWindow.cpp]: See below.

File: src/ui/w_MainWindow.cpp [hunk 4]
```diff
@@ -99,9 +99,7 @@ void MainWindow::SortConnectionList(MW_ITEM_COL byCol, bool asending)
 {
     connectionListWidget->sortByColumn(MW_ITEM_COL_NAME, Qt::AscendingOrder);
     for (auto i = 0; i < connectionListWidget->topLevelItemCount(); i++)
-    {
-        connectionListWidget->topLevelItem(i)->sortChildren(byCol, asending ? Qt::AscendingOrder : Qt::DescendingOrder);
-    }
+    { connectionListWidget->topLevelItem(i)->sortChildren(byCol, asending ? Qt::AscendingOrder : Qt::DescendingOrder); }
```
Comment (Qv2ray-dev) [inline src/ui/w_MainWindow.cpp]: See below.

**Comment by Qv2ray-dev** [comment]
some small changes requested.

**Comment by DuckSoft** [comment]
clang-format did that. 
i am innocent.

**Pull request merged**